### DISCUSSION TITLE
Is Isar still maintained?

### DISCUSSION THREAD

**Discussion by h-er0** [isar/isar#1604] [Help]
I really like Isar and want to know if it is maintained because isar is being incompatible with some Flutter packages like Riverpod

**Comment by vicenterusso** [comment]
You should use the [community fork](https://github.com/isar-community/isar)

**Reply by h-er0** [reply]
Thank you I will try. But does this mean that the author of isar @simc stopped working on the package?

**Reply by vicenterusso** [reply]
I can't answer that. What we know is that he is absent and not active on telegram group
